django_loci/base/admin.py

```diff
@@ -280,7 +280,11 @@ def _get_location_instance(self):
         data = self.cleaned_data
         location = data.get('location') or self.location_model()
         location.type = data.get('type') or location.type
-        location.is_mobile = data.get('is_mobile') or location.is_mobile
+        location.is_mobile = (
+            location.is_mobile
+            if data.get('is_mobile') is None
+            else data.get('is_mobile')
+        )
```

Comment:
Hate to be late here but there is a simpler solution:
Checkout this [best practice](http://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html).

We can do: `data.get('is_mobile', location.is_mobile)` for the same result.


Please test if this is practical in this particular place. (I haven't tested, just thought of using `default` in the `.get()` :smile: )

Reply:
Done. Works as expected. Thank you :smile: